lib/matplotlib/tests/test_figure.py

```diff
@@ -102,6 +103,38 @@ def test_align_labels_stray_axes():
     np.testing.assert_allclose(yn[::2], yn[1::2])
 
 
+## TODO add image comparison
+def test_align_titles():
+    fig, axs = plt.subplots(2, 2,
+                            subplot_kw={"xlabel": "x", "ylabel": "",
+                                        "title": "Title"})
+    axs[0][0].imshow(plt.np.zeros((5, 3)))
+    axs[0][1].imshow(plt.np.zeros((3, 5)))
+    axs[1][0].imshow(plt.np.zeros((2, 1)))
+    axs[1][1].imshow(plt.np.zeros((1, 2)))
+
+    axs[0][1].set_title('Title2', loc="left")
+    fig.align_titles()
+    fig.savefig("./result_images/test_figure/figure_align_titles")
+    compare_images("./lib/matplotlib/tests/baseline_images/figure_align_titles.png",
+                   "./result_images/test_figure/figure_align_titles.png", 0)
+
+
+## TODO add image comparison
+@image_comparison(['figure_align_titles_param'], extensions=['png', 'svg'],
```

Comment:
The test failures that are directly related to this PR are due to the baseline images for this image comparison not being added to the repo

If it is passing locally, those two files (png, svg) should be added in the `baseline_images` folder